### ISSUE TITLE
The tag jekyllgram is not a recognized Liquid tag

### ISSUE THREAD

**Issue by iamarya2k24** [benbarber/jekyll-instagram#1]
Hey @benbarber ,
I hope your doing well, I recently tried this plugin to integrate the IG feed to my blog. 
But when I try to install the plugin I get the following output.
Can you look into it? Is it due to some misconfiguration ? I have followed your instructions and added placed the file at appropriate location and added it to gem file.
Here is the error log, It might help you.

```
The page build failed for the `master` branch with the following error:

The tag `jekyllgram` on line 1 in `/_layouts/ig.html` is not a recognized Liquid tag.

Could not find gem 'jekyllgram' in any of the gem sources listed in your Gemfile
or available on this machine.

```
Thanks and Regards!
Navneet

**Comment by benbarber** [comment]
Hi, you should just need to copy the file `jekyllgram.rb` into your projects _plugins folder and then the Liquid template tags will be available.

 'jekyllgram' should not be listed in your Gemfile as it's not available as a Gem which is one reason for your build error.

**Comment by iamarya2k24** [comment]
Oh! Okay I have removed it. It has fixed the issue 👍 . But I must be doing something wrong as the Instagram grid is not showing up on pages.  I have added required variables to jekyllgram.rb and there are no errors too.
Thanks for helping out but can you point out what is causing it to not show up? :/ 

Test page with _Jekyllgram_ embed.
```html
---
layout: default
title: "Test Page with IG posts"
permalink: test.html
---
 
<body>
 
  {% jekyllgram 6 %}
  <a href="{{ photo.link }}" title="{{ photo.caption.text }}">
    <img src="{{ photo.images.thumbnail.url }}" title="{{ photo.caption.text }}" />
  </a>
{% endjekyllgram %}

</body>
```

Here is my _log_.
```
$ ./serve.sh --host=0.0.0.0
Configuration file: /home/nki/Documents/gitea-projects/navneet-suresh.github.io/_config.yml
       Deprecation: The 'gems' configuration option has been renamed to 'plugins'. Please update your config file accordingly.
            Source: /home/nki/Documents/gitea-projects/navneet-suresh.github.io
       Destination: /home/nki/Documents/gitea-projects/navneet-suresh.github.io/_site
 Incremental build: enabled
      Generating...
                    done in 1.097 seconds.
 Auto-regeneration: enabled for '/home/nki/Documents/gitea-projects/navneet-suresh.github.io'
    Server address: http://0.0.0.0:4000/
  Server running... press ctrl-c to stop.
      Regenerating: 3 file(s) changed at 2018-03-19 10:20:16
                    404.html
                    test.html
                    _plugins/jekyllgram.rb
                    ...done in 0.560404715 seconds.

      Regenerating: 3 file(s) changed at 2018-03-19 10:21:19
                    404.html
                    test.html
                    _plugins/jekyllgram.rb
                    ...done in 0.530337543 seconds.

      Regenerating: 3 file(s) changed at 2018-03-19 10:23:06
                    404.html
                    test.html
                    _plugins/jekyllgram.rb
                    ...done in 0.536404326 seconds.

```

Regards,
Navneet

**Comment by iamarya2k24** [comment]
I am attaching the link to my commit.
[Link to the repo with specific commit](https://github.com/Navneet-Suresh/navneet-suresh.github.io/tree/1b87ad2c7c5ff5a38bfd42f65762d7c1291b5a17)

**Comment by benbarber** [comment]
Did you register an Instagram app to get your Instagram user ID and API Key, then secondly you will need to make these available in the build environment as 'JEKYLLGRAM_USER' and 'JEKYLLGRAM_KEY' so the plugin can connect to you the Instagram API to get your photos.

**Comment by benbarber** [comment]
I will update this plugin in the next few days with some better instructions and improved error handling.  The plugin works well and is currently in use in the footer of my site https://benbarber.uk/

**Comment by iamarya2k24** [comment]
`Did you register an Instagram app to get your Instagram user ID and API Key, then secondly you will need to make these available in the build environment as 'JEKYLLGRAM_USER' and 'JEKYLLGRAM_KEY' so the plugin can connect to you the Instagram API to get your photos.`
@benbarber Yes I inserted the Instagram User-ID and Instagram API key in the Jekyllgram file.
 Yes, I saw your site and know that the plugin was working fine for you, and that's the reason I created this post, I was unable to replicate what you did :/ Sorry for the trouble. 
Do I need to provide environment value in _config.yml file?

**Comment by benbarber** [comment]
It turns out that Instagram has changed the API that the plugin used to not support requests with just a client id, give me some time and I will update the plugin with the required changes.

```
The Instagram API requires an access_token from authenticated users for each endpoint. 
We no longer support making requests using just the client_id.
https://www.instagram.com/developer/endpoints/
```

**Comment by iamarya2k24** [comment]
Oh, Okay!

**Comment by benbarber** [comment]
Closed by #3
